src/elements/content-sidebar/activity-feed/common/delete-confirmation/DeleteConfirmation.scss

```diff
@@ -15,6 +15,10 @@
     margin-bottom: 20px;
 }
 
+.bcs-DeleteConfirmation-btn-container {
+    display: flex;
```

Comment:
Can you check this in a language with long strings like French or German to make sure the buttons wrap as a block and align correctly when that happens?

Reply:
Sure! Just to clarify, the correct behavior for long text is to have the button become taller and the text to wrap around, is that right? (while still maintaining the side by side alignment)

Reply:
I think it's supposed to wrap to two lines, keeping the normal button height

Reply:
Ahh I see, so the entire button wraps around, I wonder if that's why the bug originally appeared...

Reply:
```suggestion
    display: flex;
    flex-wrap: wrap;
```

<img width="345" alt="Screen Shot 2021-12-10 at 1 19 45 PM" src="https://user-images.githubusercontent.com/1571667/145642647-8425b6c3-c919-49b3-a6e9-ff289443d0cd.png">
<img width="326" alt="Screen Shot 2021-12-10 at 1 18 31 PM" src="https://user-images.githubusercontent.com/1571667/145642651-b7a85fcd-5df9-4739-8f8c-75aa279de184.png">

Reply:
Thank you for the image and suggestion! I actually found the root cause of the bug, turned out there was already some styling fixes in the repo, but they were being overwritten so I fixed that! :)

Reply:
**Example of German buttons:**
<img width="1680" alt="Screen Shot 2021-12-10 at 12 39 21 PM" src="https://user-images.githubusercontent.com/95440409/145646241-8ebf33bb-0a91-4331-abd3-1d7a951c2c4b.png">

**Example of Albanian buttons:**
<img width="1680" alt="Screen Shot 2021-12-10 at 1 55 23 PM" src="https://user-images.githubusercontent.com/95440409/145646404-da0c297e-cba3-4f71-9048-9b2f70b2c5c7.png">